src/components/Common/seoLinksContainer.js

```diff
@@ -15,8 +16,8 @@ const StyledLinksColumn = styled.div`
   `}
 `
 
-const SeoLinksColumn = ({ speciality: { title, items } }, index) => (
-  <Col width={[1, 1, 1, 1, 1 / 2, 3 / 12]} key={index}>
+const SeoLinksColumn = ({ speciality: { title, items }, index }) => (
```

Comment:
Good call. I didnt see that I missed that 👍

Reply:
a few errors in the console, easy to miss!